toml.md

```diff
@@ -34,12 +34,17 @@ should be easy to parse into data structures in a wide variety of languages.
 - [MIME Type](#user-content-mime-type)
 - [ABNF Grammar](#user-content-abnf-grammar)
 
-## Spec
+## Preliminaries
+
+A TOML file must be a valid UTF-8 encoded Unicode document. Specifically this
+means that, should a file as a whole not form a
+[well-formed code-unit sequence](https://unicode.org/glossary/#well_formed_code_unit_sequence),
+the file must be rejected (preferably) or ill-formed byte sequences must be
+replaced with U+FFFD as per the Unicode specification.
```

Comment:
How about:

```
- A TOML file must be a valid UTF-8 encoded Unicode document.

  Specifically this means that a file _as a whole_ must form a [well-formed code-unit sequence](https://unicode.org/glossary/#well_formed_code_unit_sequence) and will be rejected (preferably) or have ill-formed byte sequences replaced with U+FFFD as per the Unicode specification.
```

Making the above content the final bullet in the list below?

Reply:
@pradyunsg Sure, that's perfectly fine.